packages/playground/glob-import/index.html

```diff
@@ -52,3 +53,22 @@
     2
   )
 </script>
+
+<script type="module">
+  // issue #7307
```

Comment:
A micro nitpick from me 😬 I don't think we need to put an issue number here (and in other parts of the code in this PR) as this should initially just work, but seems like an edge case we didn't catch. The Git history should have this PR linked too.

Reply:
Good feedback, unnecessary comments should be removed.